modules/UI/recording/Recording.js

```diff
@@ -497,7 +419,7 @@ const Recording = {
         case JitsiRecordingStatus.AVAILABLE:
         case JitsiRecordingStatus.OFF: {
             if (this.recordingType === 'jibri') {
-                _requestLiveStreamId()
+                _requestLiveStreamID()
```

Comment:
i like the other capitalization style as it doesn't mess with the look of the camel case when it isn't the last word.  e.g. `getRtpXmppId` is better than `getRTPXMPPID`